src/tests/seo.test.js

```diff
@@ -0,0 +1,89 @@
+import { test, expect } from "@playwright/test";
+import data from "../routes/index.js";
+
+test.describe("Testing SEO and meta tags", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/");
+  });
+
+  test("Loads the title", async ({ page }) => {
+    await expect(page).toHaveTitle(data?.title);
```

Comment:
please use the exact text expected in the result. for example, `await expect(page).toHaveTitle("Hello World");`